### ISSUE TITLE
Plex changed name string in JSON response for Synology DSM 6 - Update needed

### ISSUE THREAD

**Issue by thorli** [cowboy/synology-update-plex#52]
Hi cowboy, 
i just recognized that the script was failing lately and i got stuck on an older release.

I've seen in the logs that the version check is failing and after adding some additional debug info to the function "set_available_version()" and comparing it with the "debug_json" output i saw that the version string for Synology NAS on DSM Version 6 has changed from "Synology" to "Synology (DSM 6)"

Please see the output below for reference. 
I fixed it locally by just updating the "json_dsm_pattern" variable in the function "retrieve_dsm_version()".

Cheers..

### Script output
```
Checking for a Plex Media Server update...

[ Retrieving DSM version ]
Found DSM version <6

[ Retrieving version data ]
FAIL: Unable to retrieve version data (no version found for pattern 'Synology')

[ Debugging info ]
script_args=()
script_version=v1.1.0
dsm_major_version=6
available_version=null

```

### NAS Model
DSM1512+

### DSM Version
DSM 6.2.4-25556 Update 7

### Additional context
JSON excerpt from the relevant section:

   "nas": {
        "Synology (DSM 6)": {
            "id": "synology",
            "name": "Synology (DSM 6)",
            "release_date": 1727090209,
            "version": "1.41.0.8994-f2c27da23",
            "requirements": "Note: <b>Requires DSM 6.0 minimum to install.</b>",

**Comment by Unr3aL** [comment]
See the following pull request to make it work with all versions: https://github.com/cowboy/synology-update-plex/pull/56

I took the liberty and integrated the DSM 6 fix https://github.com/cowboy/synology-update-plex/pull/53 by @bedlam-au into my pull request.

**Comment by cowboy** [comment]
Closed by #56 and released as https://github.com/cowboy/synology-update-plex/releases/tag/v1.2.0
